projects/schematics/src/migrations/2_0/constructor-deprecations/constructor-deprecation-data.ts

```diff
@@ -21,6 +21,8 @@ import { CHECKOUT_SERVICE_MIGRATION } from './data/checkout.service.migration';
 import { CONFIGURABLE_ROUTES_SERVICE_MIGRATION } from './data/configurable-routes.service.migration';
 import { CURRENT_PRODUCT_SERVICE_MIGRATION } from './data/current-product-service.migration';
 import { EXTERNAL_ROUTES_SERVICE_MIGRATION } from './data/external-routes.service.migration';
+import { LOGIN_FORM_COMPONENT_MIGRATION } from './data/login-form.component.migration';
```

Comment:
I don't understand the notion of the `LOGIN_FORM_COMPONENT_MIGRATION` and `REGISTER_COMPONENT_MIGRATIONS` in this file. Seems from another PR.

Reply:
Yup, as mentioned in the other PR, it got moved due to the new prettier as its the import

line 41 and line 46 (I don't use those in my PR also)